src/auth/providers.js

```diff
@@ -331,7 +361,8 @@ function createProviders(auth, options = {}) {
     google: googleProvider(auth),
     facebook: facebookProvider(auth),
     mongodbCloud: mongodbCloudProvider(auth),
-    userpass: userPassProvider(auth)
+    userpass: userPassProvider(auth),
+    custom: customProvider(auth)
```

Comment:
[q] Maybe this should be called customAuth or is that too repetitive?

Reply:
Seems redundant and goes against the existing paradigm of not using `Auth` in the name